code/modules/modular_computers/file_system/programs/system/file_browser.dm

```diff
@@ -60,21 +91,40 @@
 		if(!HDD)
 			return 1
 		var/datum/computer_file/F = HDD.find_file_by_name(href_list["PRG_clone"])
+		var/checkpass = ""
+		var/currentpass = F.password
 		if(!F || !istype(F))
 			return 1
 		var/datum/computer_file/C = F.clone(1)
-		HDD.store_file(C)
+		if(!currentpass)
```

Comment:
If you opened file, you already know password, so closing it shouldn't ask password too.

Reply:
This is for cloning the file, not closing it.

Reply:
This is cloning the file not closing it.